lib/ovirt/template.rb

```diff
@@ -152,9 +152,7 @@ def build_clone_xml(options)
             end
           end
 
-          xml.os(:type => options[:os_type] || os_type) do
-            xml.boot(:dev => 'hd')
-          end
+          xml.os(:type => options[:os_type] || os_type)
```

Comment:
@matobet When provisioning PXE or ISO, I believe we always clone from the blank template then modify based on the selected source.  I think we still have the same problems in these cases.